pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumers.java

```diff
@@ -63,7 +63,7 @@
 
     /**
      * When a consumer joins, it will be added to this map with the current read position.
-     * This means that, in order to preserve ordering, new consumers can only receive old
+     * This means that, in order to preserve ordering, new consumers can only receive new
```

Comment:
The new consumer can't receive new messages if the read position when joining is greater than the mark delete position.